content/transactions/checkcreate.md

```diff
@@ -0,0 +1,41 @@
+## CheckCreate
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/app/tx/impl/CreateCheck.cpp "Source")
+
+_Requires the [Checks Amendment](reference-amendments.html#checks)._
+
+Create a Check object in the ledger, which is a deferred payment that can be cashed by its intended destination. The sender if this transaction is the sender of the Check.
+
+Example CheckCreate:
+
+```json
+{
+  "TransactionType": "CheckCreate",
+  "Account": "rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo",
+  "Destination": "rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy",
+  "SendMax": "100000000",
+  "Expiration": 570113521,
+  "InvoiceID": "6F1DFD1D0FE8A32E40E1F2C05CF1C15545BAB56B617F9C6C2D63A6B704BEF59B",
+  "DestinationTag": 1,
+  "Fee": "12"
+}
+```
+
+In addition to the [common fields](#common-fields), a CheckCancel transaction has the following:
+
+| Field            | JSON Type           | [Internal Type][] | Description     |
+|:-----------------|:--------------------|:------------------|:----------------|
+| `Destination`    | String              | Account           | The unique address of the [account](concept-accounts.html) that can cash the check. |
+| `SendMax`        | [Currency Amount][] | Amount            | Maximum amount of source currency the Check is allowed to debit the sender, including [transfer fees](concept-transfer-fees.html) on non-XRP currencies. The Check can only credit the destination with the same currency (from the same issuer, for non-XRP currencies). For non-XRP amounts, the nested field names MUST be lower-case. |
+| `Expiration`     | Unsigned Integer    | UInt32            | _(Optional)_ Time after which the check is no longer valid, in [seconds since the Ripple Epoch](reference-rippled.html#specifying-time). |
+| `DestinationTag` | Unsigned Integer    | UInt32            | _(Optional)_ Arbitrary tag that identifies the reason for the check, or a hosted recipient to pay. |
+| `InvoiceID`      | String              | Hash256           | _(Optional)_ Arbitrary 256-bit hash representing a specific reason or identifier for this check. |
```

Comment:
As mentioned above, I'd order these table fields in parallel with either the JSON example or alphabetically (I know they come back in random order due to their hash structure, but it's nice for the reader).

Reply:
Reordered them to be required fields (alphabetical) then optional fields (alphabetical)